src/locales/en/tdp.json

```
@@ -42,6 +42,12 @@
       "startTour": "Start Tour",
       "tourLevelBeginner": "General introduction",
       "tourLevelAdvanced": "Advanced tours"
+    },
+    "loginMenu": {
+      "automaticLoginDisclaimer": "You will be automatically logged into Ordino as soon as you visit the site. If your access is expired, please refresh the page to log in again.",
+      "modalTitle": "Login to Ordino",
+      "acceptGenieTerms": "Accept GENIE terms checkbox in login dialog is missing!",
+      "refreshPage": "Refresh Page"
```

```suggestion
      "refreshPage": "Refresh page"
```
Please use sentence-case here.